styles/mobile.css

```diff
@@ -105,5 +110,123 @@ ul li {
 }
 
 .social-icon {
-  padding: 28px;
+  padding: 28px 28px 0 28px;
+}
+
+.down-items-container {
+  display: flex;
+  color: #fff;
+  justify-content: center;
+  align-items: center;
+}
+
+.down-text {
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #a2a3ae;
+  margin: 0 12px;
+}
+
+/* STYLING PROJECTS SECTION */
+
+.projects-container {
+  display: grid;
+}
+
+.projects-heading {
+  color: #9490ff;
+  text-align: left;
+  font-size: 30px;
+  line-height: 62px;
+  margin: 24px;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  margin-top: 48px;
+  grid-column-start: 1;
+  grid-column-end: 5;
+}
+
+.card-img {
+  text-align: center;
+}
+
+.card-title {
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin: -24px 24px;
+}
+
+.card-text {
+  color: #dcdce8;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 24px 24px;
+}
+
+.card-skills-list > ul {
+  display: flex;
+  margin-top: -24px;
+}
+
+.card-skills-list > ul > li {
+  padding: 2rem;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.skill-divider {
+  margin-left: 1rem;
+}
+
+.card-btn {
+  background-color: #7d73fc;
+  width: 89px;
+  height: 48px;
+  margin: -24px 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #edecff;
+  font-size: small;
+  letter-spacing: 3%;
+  border: 0;
+  z-index: 1;
+}
+
+.card-btn:hover {
+  background-color: #d5d3ee !important;
+  color: #fff;
+}
```

Comment:
- [ ] Please consider sticking to the design template which you choose. currently, your button's hover and pressed state are not the same as your chosen template. Could you consider fixing that?

figma Template:
<img width="180" alt="chrome_xIrxzejnHg" src="https://user-images.githubusercontent.com/81468675/143440994-35c9b649-3554-40eb-9761-46917b1e680d.png">